docs/helm/helm.md

```diff
@@ -45,15 +45,21 @@ spec:
 | metadata.name |   | Helm Chart name | NAME |
 | spec.chart |   | Helm Chart name in repository, or complete HTTPS URL to chart archive (.tgz) | CHART |
 | spec.targetNamespace | default | Helm Chart target namespace | `--namespace` |
+| spec.createNamespace | false | Create target namespace if not present | `--create-namespace` |
 | spec.version |   | Helm Chart version (when installing from repository) | `--version` |
 | spec.repo |   | Helm Chart repository URL | `--repo` |
 | spec.repoCA | | Specify the certificates of HTTPS-enabled servers | `--ca-file` |
+| spec.repoCAConfigMap |  | Values of all keys in ConfigMap will be merged with repoCA | `--ca-file` |
```

Comment:
```suggestion
| spec.repoCA | | Verify certificates of HTTPS-enabled servers using this CA bundle. Should be a string containing one or more PEM-encoded CA Certificates. | `--ca-file` |
| spec.repoCAConfigMap | | Reference to a ConfigMap containing CA Certificates to be be trusted by Helm. Can be used along with or instead of `repoCA` | `--ca-file` |
```